tools/pipelines/test-coverage.yml

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# test-coverage pipeline (for the purpose of detecting flaky tests)
+
+name: $(Build.BuildId)
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "00 03 * * SAT"
+    displayName: Schedlued Tests on Saturday Nights Weekly
+    branches:
+      include:
+      - main
+      - next
+      - release/*
```

Comment:
Do we want to test release branches?  even if some older release branch never have any changes? I wonder whether we should skip the release branches.

Reply:
I'm inclined to skip. I don't see much value from it.

Reply:
yes I have removed that